android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.java

```diff
@@ -128,18 +128,19 @@ public void setDRM(final ReactExoplayerView videoView, @Nullable ReadableMap drm
         if (drm != null && drm.hasKey(PROP_DRM_TYPE)) {
             String drmType = ReactBridgeUtils.safeGetString(drm, PROP_DRM_TYPE);
             String drmLicenseServer = ReactBridgeUtils.safeGetString(drm, PROP_DRM_LICENSESERVER);
-            ReadableMap drmHeaders = ReactBridgeUtils.safeGetMap(drm, PROP_DRM_HEADERS);
+            ReadableArray drmHeadersArray = (drm.hasKey(PROP_DRM_HEADERS)) ? drm.getArray(PROP_DRM_HEADERS) : null;
```

Comment:
you should be able to use safeGetArray here :)

Reply:
That's a good point. I'll try to fix them.

Reply:
Cool
Have a check on this commit https://github.com/react-native-video/react-native-video/pull/3487/commits/e69666d2458b262e52d2ba01775426a8e224cf4b